tests/utils/zephyr/test_coordinate_systems.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2025 D-Wave
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# ================================================================================================
+
+
+import unittest
+
+from parameterized import parameterized
+
+from minorminer.utils.zephyr.coordinate_systems import (
+    CartesianCoord,
+    ZephyrCoord,
+    cartesian_to_zephyr,
+    zephyr_to_cartesian,
+)
+
+
+class TestCoordinateSystems(unittest.TestCase):
+
+    def test_cartesian_to_zephyr_runs(self):
+        xyks = [(0, 1), (1, 0, None), (12, -3)]
+        ccoords = [CartesianCoord(*xyk) for xyk in xyks]
+        for ccoord in ccoords:
+            cartesian_to_zephyr(ccoord=ccoord)
```

Comment:
I would fold the two `*_runs` tests with the parameterized ones, but if you want to keep them as smoke tests, I recommend at least testing the returned type:

```suggestion
            self.assertIsInstance(cartesian_to_zephyr(ccoord=ccoord), ZephyrCoord)
```